tsconfig.shared.json

```diff
@@ -28,8 +28,9 @@
     "pretty": true,
     "sourceMap": true,
     "typeRoots": ["./typings", "./node_modules/@types"],
-    "types": []
+    "types": ["node", "jest"]
   },
+  "include": ["typings/**/*"],
   "exclude": [
```

Comment:
Excluding `"**/test/**",` here means the type checking with `ts-jest` does not work.

Could change to `"**/test/**/*.js",` so TS files are included in the type checking.